vtex-apps/store-components
# Feature/product images slider per group prop

#### What problem is this solving?

This solution add the possibility of change the number of the thumbnail slides skipped when the thumbnail arrow is clicked.

#### How to test it?

Click on the link below to see the implementation. The value of the slides skipped are set by the block property "slidesPerGroup". If the prop isnt used, the default value of 4 slides is set. The block documentation was updated with the description of the property added.

https://avtjoao2--avantivtexio.myvtex.com/jogo-de-banho-padrao-fantastique-03-02-0072-p41525-copy42/p

#### Screenshots or example usage:
The value of the slides skipped used in the workspace was 1

![image](https://user-images.githubusercontent.com/88465655/128778932-a227c22f-47f8-4ef4-84b6-ec1d2d749b70.png)



#### Describe alternatives you've considered, if any.



#### Related to / Depends on



#### How does this PR make you feel? [:link:](http://giphy.com/)



![](put .gif link here - can be found under "advanced" on giphy)
